routes/web.php

```diff
@@ -90,37 +94,40 @@ protected function svg()
                     }
                 }
             }
-            
+
             if (!empty($path)) {
                 $svg .= '<path class="qr-' . $M_TYPE . ' ' . $this->options->cssClass . '" stroke="transparent" fill="' . $value . '" fill-opacity="' . $this->options->svgOpacity . '" d="' . $path . '" />' . $circle;
             }
-            
+
         }
-        
+
         // close svg
         $svg .= '</g>
 <image xlink:href="circle.png" width="'.$length.'"/>
 <!--<image xlink:href="eye.png" x="' .($length/2-21). '" y="' .($length/2-21). '" width="42px"/>-->
 </svg>' . $this->options->eol;
-        
+
         // if saving to file, append the correct headers
         if ($this->options->cachefile) {
             return '<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">' . $this->options->eol . $svg;
         }
-        
+
         return $svg;
     }
 }
 
 class MyCustomOutput extends QROutputAbstract
```

Comment:
- Each class must be in a file by itself
- Each class must be in a namespace of at least one level (a top-level vendor name)